spec/system/case_contacts/new_spec.rb

```diff
@@ -1,87 +1,119 @@
 require "rails_helper"
 require "action_view"
 
-RSpec.describe "case_contacts/new", type: :system, js: true, flipper: true do
+RSpec.describe "case_contacts/new", type: :system, js: true do
```

Comment:
I went ham on this spec to better understand the wizard, and there was a lot of extra setup or examples that weren't testing what they promised.  So it's a big diff!